nodes/scenenet_to_rosbag.py

```diff
@@ -407,6 +413,28 @@ def publish(scenenet_path, trajectory, output_bag, to_frame):
     args = parser.parse_args()
     if args.scenenet_path:
         scenenet_path = args.scenenet_path
+    if args.protobuf_path:
+        protobuf_path = args.protobuf_path
+    else:
+        # Default
+        protobuf_path = os.path.join(scenenet_path, 'data/scenenet_rgbd_val.pb')
+        print("Path to protobuf file not found as an argument (use "
+              "-protobuf_path to indicate one). Using default value "
+              "{}".format(protobuf_path))
```

Comment:
Is this message necessary? If not, just stick to the _default_ field of the ArgumentParser as shown above, to be consistent with the rest of the arguments handling, and remove this whole _else_ code.

Reply:
Thank you for the suggestion, fixed in 71d7f1818973bc2ca80fda9e8005984091c38fdf.